.circleci/config.yml

```diff
@@ -547,6 +546,7 @@ commands:
 
 jobs:
   build-docs:
+    # TODO this now depends on build-proto
```

Comment:
there is no circleci feature to declare this dependency here is there?

Reply:
I'd drop all of the circleci changes and just let proto rebuild.  It's just a couple of seconds, right?